backend/program/symlink.py

```diff
@@ -10,6 +10,7 @@
 class SymlinkConfig(BaseModel):
     host_path: Path
     container_path: Path
+    symlink_path: Path = None

```

Comment:
why a 3rd path? 

The `container_path` is the path that plex sees. Here's an example, 

![image](https://github.com/dreulavelle/iceberg/assets/5782630/f07cd5ed-de75-4f2b-870c-dc5d958c37f1)

my paths are `/mnt/zurg` for host_path and `/mnt/zurg` for container_path since I set plex volume mount to `/mnt:/mnt`. container_path is the path that Plex see's to be able to read the symlink, and host_path is the rclone path that symlinks get pointed to. Think of it like this,

```
source = container_path
destination = host_path
```

Reply:
If we need to change path names to make it more readable/understandable, we can change:

`host_path` -> `rclone_path`
`container_path` -> `symlink_path`

And possibly add a brief description in the frontend about it to users?

Reply:
@AyushSehrawat I explain it here